sigstore-java/src/main/java/dev/sigstore/http/GrpcChannels.java

```diff
@@ -39,4 +39,23 @@ public static ManagedChannel newManagedChannel(URI serverUrl, HttpParams httpPar
     }
     return channelBuilder.build();
   }
+
+  /**
+   * Create a new managed channel, this may be reused across multiple requests to a host, and must
+   * be closed when finished.
+   *
+   * @param serverUrl the host to connect to
+   * @param httpParams the http configuration
+   * @return a reusable grpc channel
+   */
+  public static ManagedChannel newManagedChannel(String serverUrl, HttpParams httpParams) {
```

Comment:
when we extract the "authority" from a uri (Strip the "http...") it returns a string, so we're going to greate managed channels with a string.